openapiart/common.py

```diff
@@ -423,14 +423,25 @@ def validate_hex(self, hex):
     def validate_integer(self, value, min, max, type_format=None):
         if value is None or not isinstance(value, int):
             return False
-        if type_format == "uint32" and value < 0:
-            return False
-        if type_format == "uint64" and value < 0:
-            return False
         if min is not None and value < min:
             return False
         if max is not None and value > max:
             return False
+        if type_format is not None:
```

Comment:
if format is not mentioned, it should be assumed to be int32 - don't see any code for checking that (and the min-max range)

Reply:
added it here in generator.py line number 1754

Reply:
set default `type_format` to `int32`